### PR TITLE
Fix bug in last error "converting NULL to string is unsupported"

### PR DESCRIPTION
**Pull request by vvxxvvxx**
**Description**

Changes proposed in this pull request:

- Fixed the bug in PR#1405 to set default value of last error fields to ''.

**Related issue(s)**

### CODE REVIEW

File: components/schema-migrator/migrations/provisioner/202203171600_operation_alter_last_error.up.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE operation
```
Comment (koala7659) [inline components/schema-migrator/migrations/provisioner/202203171600_operation_alter_last_error.up.sql:1]: If you change default values in migration script - you should also update them in provisioner/assets/database/provisioner.sql file

**Review by koala7659 — CHANGES_REQUESTED** [review]

File: components/schema-migrator/migrations/provisioner/202203171600_operation_alter_last_error.up.sql (hunk 1 above)
Comment (vvxxvvxx) [reply components/schema-migrator/migrations/provisioner/202203171600_operation_alter_last_error.up.sql:1]: added `NOT NULL` in provisioner.sql for last error fields..

**Review by koala7659 — CHANGES_REQUESTED** [review]
There should be a respective step down added in database migration

**Comment by vvxxvvxx** [comment]
> There should be a respective step down added in database migration

this is the addition to`202202241500_operation_last_error.up.sql`.
Is `202202241500_operation_last_error.down.sql` enough for the down? this will remove the columns added.

**Comment by vvxxvvxx** [comment]
> There should be a respective step down added in database migration

Added the down step.

**Comment by vvxxvvxx** [comment]
huskies are looking into the error in `pre-main-control-plane-reconciler-upgrade-kyma2-latest-to-main-gardener`

**Review by Maladie — APPROVED** [review]

**Review by koala7659 — APPROVED** [review]
LGTM

**Pull request merged**